### PR TITLE
Fix bug and add support for integer resnames which exist in PDB

### PR DESCRIPTION
**Pull request by stefdoerr**
Currently if you add a residue with an integer name to the XML definitions it will crash because it will convert the resname to a float number which will not match any class definitions.

https://www.rcsb.org/structure/5VBL
Here for example you have residue 200 (4-CHLORO-L-PHENYLALANINE)
https://www.rcsb.org/ligand/200

This PR fixes that issue by insisting on string type for residue names even if they can be converted to float.

### CODE REVIEW

**Comment by stefdoerr** [comment]
If you approve this, would be awesome if you can make quickly a bug-fix release as well so that I can get rid of my really ugly workaround as well :smiley:

**Review by sobolevnrm — APPROVED** [review]

**Review by intendo — APPROVED** [review]

**Comment by sobolevnrm** [comment]
https://pypi.org/project/pdb2pqr/3.5.1/

**Comment by stefdoerr** [comment]
Ah, the conda-forge recipe is not maintained by you I see. Might make sense though to make a release since 3.5.0 was also not released there yet. Should I ask the maintainer over there?
Thanks!

**Pull request merged**
